runtime/datastructures/src/decls.rs

```diff
@@ -98,6 +210,7 @@ extern "C" {
   pub fn move_int(result: *mut Int) -> *mut Int;
   pub fn printConfigurationInternal(file: *mut FILE, subject: *const Block, sort: *const c_char);
   pub fn hook_KEQUAL_eq(k1: K, k2: K) -> bool;
+  pub fn hook_KORD_cmp(k1: K, k2: K) -> i64;
```

Comment:
You should still call it hook_KEQUAL_cmp. That's much more in alignment with how hooks are named in K and means we can eventually add it to the prelude once we get MInt working.

Reply:
Done